### PR TITLE
add upgrade-examples app with v8 and v9 working together

### PR DESCRIPTION
**Pull request by micahgodbolt**
Add a new app to house the v8 to v9 upgrade examples. If this works well, I'll probably create a similar app for the v0 to v9. 

fixes https://github.com/microsoft/fluentui-react-upgrade/issues/6

### CODE REVIEW

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react-components`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>Avatar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/Avatar-mount_base.html">1278</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/Avatar-mount.html">1263</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Button</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/Button-mount_base.html">758</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/Button-mount.html">751</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProvider-mount_base.html">2100</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProvider-mount.html">2101</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProviderWithTheme-mount_base.html">389</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProviderWithTheme-mount.html">374</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender_base.html">348</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender.html">313</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>FluentProviderWithTheme</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount_base.html">408</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/FluentProviderWithTheme-virtual-rerender-with-unmount.html">391</a></td>
            <td>10</td>
            <td></td>
           </tr>
<tr>
            <td>MakeStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/MakeStyles-mount_base.html">1904</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test-react-components/MakeStyles-mount.html">1917</a></td>
            <td>50000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9425"/>

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<details>
<summary>Unchanged fixtures</summary>

| Package & Exports                                                                                                                                                             |          Size (minified/GZIP) |
| ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------------------: |
| <samp>priority-overflow</samp> <br /> <abbr title='bundle-size/overflow-manager.fixture.js'>createOverflowManager</abbr>                                                      |    `2.836 kB`<br />`1.209 kB` |
| <samp>react-accordion</samp> <br /> <abbr title='bundle-size/Accordion.fixture.js'>Accordion (including children components)</abbr>                                           |  `74.792 kB`<br />`22.516 kB` |
| <samp>react-avatar</samp> <br /> <abbr title='bundle-size/Avatar.fixture.js'>Avatar</abbr>                                                                                    |  `45.142 kB`<br />`13.111 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/Badge.fixture.js'>Badge</abbr>                                                                                       |   `20.895 kB`<br />`6.567 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/CounterBadge.fixture.js'>CounterBadge</abbr>                                                                         |   `21.848 kB`<br />`6.883 kB` |
| <samp>react-badge</samp> <br /> <abbr title='bundle-size/PresenseBadge.fixture.js'>PresenceBadge</abbr>                                                                       |   `21.951 kB`<br />`6.565 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/Button.fixture.js'>Button</abbr>                                                                                    |   `28.013 kB`<br />`8.059 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/CompoundButton.fixture.js'>CompoundButton</abbr>                                                                    |   `33.508 kB`<br />`9.092 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/MenuButton.fixture.js'>MenuButton</abbr>                                                                            |   `29.796 kB`<br />`8.665 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/SplitButton.fixture.js'>SplitButton</abbr>                                                                          |   `36.268 kB`<br />`9.863 kB` |
| <samp>react-button</samp> <br /> <abbr title='bundle-size/ToggleButton.fixture.js'>ToggleButton</abbr>                                                                        |    `37.395 kB`<br />`8.68 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.All.fixture.js'>Card - All</abbr>                                                                                |  `53.619 kB`<br />`15.372 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/Card.fixture.js'>Card</abbr>                                                                                          |  `48.904 kB`<br />`14.089 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardFooter.fixture.js'>CardFooter</abbr>                                                                              |    `7.686 kB`<br />`3.264 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardHeader.fixture.js'>CardHeader</abbr>                                                                              |     `9.251 kB`<br />`3.78 kB` |
| <samp>react-card</samp> <br /> <abbr title='bundle-size/CardPreview.fixture.js'>CardPreview</abbr>                                                                            |    `7.658 kB`<br />`3.291 kB` |
| <samp>react-combobox</samp> <br /> <abbr title='bundle-size/Combobox.fixture.js'>Combobox</abbr>                                                                              |    `6.817 kB`<br />`2.901 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/MultipleComponents.fixture.js'>react-components: Accordion, Button, FluentProvider, Image, Menu, Popover</abbr> | `176.089 kB`<br />`49.113 kB` |
| <samp>react-components</samp> <br /> <abbr title='bundle-size/ProviderAndTheme.fixture.js'>react-components: FluentProvider & webLightTheme</abbr>                            |  `32.526 kB`<br />`10.645 kB` |
| <samp>react-divider</samp> <br /> <abbr title='bundle-size/Divider.fixture.js'>Divider</abbr>                                                                                 |   `15.385 kB`<br />`5.539 kB` |
| <samp>react-image</samp> <br /> <abbr title='bundle-size/Image.fixture.js'>Image</abbr>                                                                                       |   `10.109 kB`<br />`3.958 kB` |
| <samp>react-input</samp> <br /> <abbr title='bundle-size/Input.fixture.js'>Input</abbr>                                                                                       |   `21.635 kB`<br />`7.166 kB` |
| <samp>react-label</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Label</abbr>                                                                                       |    `8.371 kB`<br />`3.504 kB` |
| <samp>react-link</samp> <br /> <abbr title='bundle-size/Label.fixture.js'>Link</abbr>                                                                                         |   `11.106 kB`<br />`4.507 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.fixture.js'>Menu (including children components)</abbr>                                                          | `105.852 kB`<br />`32.433 kB` |
| <samp>react-menu</samp> <br /> <abbr title='bundle-size/Menu.Selectable.fixture.js'>Menu (including selectable components)</abbr>                                             | `109.031 kB`<br />`32.897 kB` |
| <samp>react-popover</samp> <br /> <abbr title='bundle-size/Popover.fixture.js'>Popover</abbr>                                                                                 |  `96.787 kB`<br />`29.559 kB` |
| <samp>react-portal</samp> <br /> <abbr title='bundle-size/Portal.fixture.js'>Portal</abbr>                                                                                    |     `6.272 kB`<br />`2.17 kB` |
| <samp>react-positioning</samp> <br /> <abbr title='bundle-size/usePopper.fixture.js'>usePopper</abbr>                                                                         |    `23.21 kB`<br />`8.084 kB` |
| <samp>react-priority-overflow</samp> <br /> <abbr title='bundle-size/hooks.fixture.js'>hooks only</abbr>                                                                      |   `10.606 kB`<br />`4.087 kB` |
| <samp>react-provider</samp> <br /> <abbr title='bundle-size/FluentProvider.fixture.js'>FluentProvider</abbr>                                                                  |    `14.009 kB`<br />`5.25 kB` |
| <samp>react-select</samp> <br /> <abbr title='bundle-size/Select.fixture.js'>Select</abbr>                                                                                    |    `7.853 kB`<br />`3.297 kB` |
| <samp>react-slider</samp> <br /> <abbr title='bundle-size/Slider.fixture.js'>Slider</abbr>                                                                                    |   `25.402 kB`<br />`8.251 kB` |
| <samp>react-spinner</samp> <br /> <abbr title='bundle-size/Spinner.fixture.js'>Spinner</abbr>                                                                                 |      `6.815 kB`<br />`2.9 kB` |
| <samp>react-switch</samp> <br /> <abbr title='bundle-size/Switch.fixture.js'>Switch</abbr>                                                                                    |   `24.279 kB`<br />`8.001 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Default.fixture.js'>Text - Default</abbr>                                                                        |   `10.797 kB`<br />`4.233 kB` |
| <samp>react-text</samp> <br /> <abbr title='bundle-size/Text.Wrappers.fixture.js'>Text - Wrappers</abbr>                                                                      |   `14.113 kB`<br />`4.576 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/SingleThemeToken.fixture.js'>Single theme token import</abbr>                                                        |            `69 B`<br />`89 B` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsAllThemes.fixture.js'>Teams: all themes</abbr>                                                                  |   `29.426 kB`<br />`6.551 kB` |
| <samp>react-theme</samp> <br /> <abbr title='bundle-size/TeamsLightTheme.fixture.js'>Teams: Light theme</abbr>                                                                |     `18.42 kB`<br />`5.27 kB` |
| <samp>react-tooltip</samp> <br /> <abbr title='bundle-size/Tooltip.fixture.js'>Tooltip</abbr>                                                                                 |  `42.837 kB`<br />`14.727 kB` |
| <samp>react-utilities</samp> <br /> <abbr title='bundle-size/SSRProvider.fixture.js'>SSRProvider</abbr>                                                                       |          `189 B`<br />`161 B` |

</details>
<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/f10cc045484b962652a400ef2ef4f3412c8d168b'>f10cc045484b962652a400ef2ef4f3412c8d168b</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/BaseButton-mount_base.html">791</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/BaseButton-mount.html">793</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Breadcrumb-mount_base.html">2267</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Breadcrumb-mount.html">2259</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Checkbox-mount_base.html">1269</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Checkbox-mount.html">1278</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/CheckboxBase-mount_base.html">1093</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/CheckboxBase-mount.html">1098</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ChoiceGroup-mount_base.html">4054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ChoiceGroup-mount.html">4029</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ComboBox-mount_base.html">828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ComboBox-mount.html">826</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/CommandBar-mount_base.html">8863</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/CommandBar-mount.html">8851</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ContextualMenu-mount_base.html">9981</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ContextualMenu-mount.html">10084</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DefaultButton-mount_base.html">971</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DefaultButton-mount.html">970</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DetailsRow-mount_base.html">3279</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DetailsRow-mount.html">3239</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DetailsRowFast-mount_base.html">3272</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DetailsRowFast-mount.html">3344</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DetailsRowNoStyles-mount_base.html">3083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DetailsRowNoStyles-mount.html">3074</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Dialog-mount_base.html">1930</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Dialog-mount.html">1916</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DocumentCardTitle-mount_base.html">146</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/DocumentCardTitle-mount.html">144</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Dropdown-mount_base.html">2839</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Dropdown-mount.html">2844</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/FocusTrapZone-mount_base.html">1550</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/FocusTrapZone-mount.html">1567</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/FocusZone-mount_base.html">1555</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/FocusZone-mount.html">1542</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/IconButton-mount_base.html">1543</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/IconButton-mount.html">1525</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Label-mount_base.html">303</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Label-mount.html">308</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Layer-mount_base.html">2523</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Layer-mount.html">2609</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Link-mount_base.html">399</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Link-mount.html">406</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/MenuButton-mount_base.html">1259</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/MenuButton-mount.html">1273</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/MessageBar-mount_base.html">1851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/MessageBar-mount.html">1822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Nav-mount_base.html">2842</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Nav-mount.html">2873</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/OverflowSet-mount_base.html">939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/OverflowSet-mount.html">930</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Panel-mount_base.html">1864</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Panel-mount.html">1838</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Persona-mount_base.html">714</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Persona-mount.html">737</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Pivot-mount_base.html">1259</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Pivot-mount.html">1268</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/PrimaryButton-mount_base.html">1116</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/PrimaryButton-mount.html">1128</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Rating-mount_base.html">6651</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Rating-mount.html">6641</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SearchBox-mount_base.html">1141</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SearchBox-mount.html">1138</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Shimmer-mount_base.html">2138</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Shimmer-mount.html">2136</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Slider-mount_base.html">1687</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Slider-mount.html">1667</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SpinButton-mount_base.html">4369</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SpinButton-mount.html">4335</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Spinner-mount_base.html">380</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Spinner-mount.html">369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SplitButton-mount_base.html">2739</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SplitButton-mount.html">2738</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Stack-mount_base.html">440</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Stack-mount.html">443</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/StackWithIntrinsicChildren-mount_base.html">1993</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/StackWithIntrinsicChildren-mount.html">1979</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/StackWithTextChildren-mount_base.html">4551</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/StackWithTextChildren-mount.html">4521</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SwatchColorPicker-mount_base.html">10059</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/SwatchColorPicker-mount.html">10028</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/TagPicker-mount_base.html">2335</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/TagPicker-mount.html">2381</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/TeachingBubble-mount_base.html">81766</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/TeachingBubble-mount.html">82039</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Text-mount_base.html">367</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Text-mount.html">350</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/TextField-mount_base.html">1216</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/TextField-mount.html">1205</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ThemeProvider-mount_base.html">1013</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ThemeProvider-mount.html">1017</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ThemeProvider-virtual-rerender_base.html">554</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ThemeProvider-virtual-rerender.html">561</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1601</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1593</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Toggle-mount_base.html">686</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/Toggle-mount.html">694</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/buttonNative-mount_base.html">105</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/22226/perf-test/buttonNative-mount.html">110</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Comment by ecraig12345** [comment]
Can you delete the empty changelog files?

**Comment by micahgodbolt** [comment]
> Can you delete the empty changelog files?

@ecraig12345 i can delete them if they need to be. Wasn't sure if those were typically added on package creation or not. I just left it in there but cleaned it out after duplicating the vr-test package

**Review by GeoffCoxMSFT — APPROVED** [review]

File: apps/upgrade-examples-v8-v9/package.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
+{
+  "name": "@fluentui/upgrade-examples-v8-v9",
+  "version": "1.0.0",
+  "private": true,
+  "description": "A collection of examples demonstrating how to upgrade from v8 to v9",
+  "scripts": {
+    "build": "just-scripts build",
+    "build-storybook": "build-storybook -s ./public -o ./dist/storybook --docs",
+    "chromatic": "npx chromatic@6.4.3 --project-token $CHROMATIC_PROJECT_TOKEN --exit-zero-on-changes --build-script-name build-storybook",
```
[the remark this answers is not in the document]
Comment (micahgodbolt) [reply apps/upgrade-examples-v8-v9/package.json:9]: Yes, we want to deploy it and then pull the stories into the react-components via storybook composition

File: apps/upgrade-examples-v8-v9/package.json [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    "start": "just-scripts dev:storybook"
+  },
+  "devDependencies": {
+    "@fluentui/eslint-plugin": "*"
+  },
+  "dependencies": {
+    "@fluentui/react": "^8.62.4",
+    "@fluentui/scripts": "^1.0.0",
+    "@fluentui/storybook": "^1.0.0",
+    "@fluentui/react-components": "^9.0.0-rc.2",
```
[the remark this answers is not in the document]
Comment (micahgodbolt) [reply apps/upgrade-examples-v8-v9/package.json:23]: i had to downlevel this due to another package using this version.

**Review by ecraig12345 — APPROVED** [review]

**Pull request merged**